test/common_tester.h

```diff
@@ -547,6 +549,17 @@ class CommonTester : public ::testing::Test
                       delta.hat() * delta_other.hat() - delta_other.hat() * delta.hat());
   }
 
+  void evalIdentityActPoint()
+  {
+    using Point = typename LieGroup::Vector;
+
+    Point pin = Point::Random();
+
+    Point pout = LieGroup::Identity().act(pin);
+
+    EXPECT_EIGEN_NEAR(pin, pout);
+  }
+
```

Comment:
Why not? But could we imagine a second test a little more conclusive?

I will approve the review anyway regardless of this comment

Reply:
Sure we can add a better test later on, for now I just want the `Vector` typedef to be compiled and used to assert its dimensions.
I'll merge and come back to it later.